cli/collection.go

```diff
@@ -81,7 +81,7 @@ func MakeCollectionCommand(cfg *config.Config) *cobra.Command {
 				fetchedCols := cols
 				cols = nil
 				for _, c := range fetchedCols {
-					if c.Name() == name {
+					if c.Name().Value() == name {
```

Comment:
question: Why have you skipped `HasValue()` here before calling `Value()`. If IRRC the value returned is invalid unless `HasValue() == true`

Reply:
`name` cannot be `""` here, and at the moment `fetchedCols` cannot contain any nameless collections.

The code that populates `fetchedCols` will likely need restructuring somewhat in the followup PR that sorts out the version stuff, this line will be looked at again at that point.